### ISSUE TITLE
As a member of the outreach team, I'd like to be able to share a direct link to sign up for the newsletter

### ISSUE THREAD

**Issue by awfrancisco** [18F/18f.gsa.gov#1891]
Previously, 18f.gsa.gov/#newsletter sent people to the newsletter form. Now, I have to say "Sign up at the bottom of 18f.gsa.gov."

**Comment by awfrancisco** [comment]
Temporarily fixed by #1894

**Comment by coreycaitlin** [comment]
Handing this off to @austinhernandez for thinking through...

- Should this be on its own page or part of a "Contact" page?
- What should it look like?

**Comment by austinhernandez** [comment]
![contact_newsletter](https://cloud.githubusercontent.com/assets/23194720/22561069/c2b81af4-e944-11e6-8ed8-2c87c3d76169.png)

![contact_approved](https://cloud.githubusercontent.com/assets/23194720/22561067/c06de170-e944-11e6-9301-821941ff0af3.png)

I had a little extra time to think through the "hire 18F" (maybe known in the future as the contact page) and newsletter sign up process and if there was any way to improve it. I used the content from the existing hire 18F page and used the project page template to break up content more and have important CTAs/info in a sidebar.
1. Basically just added a newsletter section to the bottom of the page. If nothing else could get done, this is the MVP.
2. I thought it would be cool to do inline validation and confirmation content to replace the input field (both in the footer and on the contact page)
3. I reorganized the contact page to use the project page layout. We could also use the about page layout if we wanted to bring in more playbook content!
4. I added some links and buttons to the meta content chunk.
